core/testsuite/src/test/java/com/blazebit/persistence/testsuite/WindowFunctionTest.java

```diff
@@ -579,6 +575,14 @@ public void testParsedNamedWindowDefinitionWithRange() {
                 .orderByAsc("per.age")
                 ;
 
+        String queryString = criteria.getQueryString();
+
+        assertEquals("SELECT per.age, window_sum(per.age,'ORDER BY',per.age,'ASC NULLS LAST', " +
```

Comment:
Not sure if we're completely consistent w.r.t. the rendering of spaces here. Shouldn't this be:

```suggestion
        assertEquals("SELECT per.age, window_sum(per.age, 'ORDER BY', per.age, 'ASC NULLS LAST', " +
```
?

Reply:
That's on purpose. It's internal, so no need for assertions about that here.